tests/integration/commands/link/link.test.ts

```diff
@@ -9,6 +9,53 @@ import { getCLIOptions, withMockApi } from '../../utils/mock-api.js'
 import { withSiteBuilder } from '../../utils/site-builder.js'
 
 describe('link command', () => {
+  test.todo('should link to matching site given `--site-id`')
+  test.todo('should print an error and exit when no site with given `--site-id` is found')
+
+  test.todo('should link to matching site given `--site-name`')
+  test.todo('should print an error and exit when no site with given `--site-name` is found')
```

Comment:
I noticed missing coverage for `link`. It felt pretty out of scope but I wanted to at least leave these breadcrumbs.